src/index.tsx

```diff
@@ -15,12 +15,24 @@ import NtpSyncComponent from 'src/ntp/NtpSyncComponent';
 import registerServiceWorker from 'src/registerServiceWorker';
 import { ApolloClientsContext } from 'src/VirtualMonitorApolloClients';
 import VirtualMonitorLocalState from 'src/VirtualMonitorLocalState';
+ import {default as config} from 'src/monitorConfig.js';
+
+var domain = window.location.hostname;
```

Comment:
use let instead of var and I think this could be const

Reply:
fixed